### PR TITLE
Fix/116/change error messages in update fact handler and tests

### PR DESCRIPTION
**Pull request by OlhaPivniuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by HlibBondarev — APPROVED** [review]
Ok

File: Streetcode/Streetcode.WebApi/Controllers/Streetcode/TextContent/FactController.cs [hunk 1]
```diff
@@ -30,22 +30,22 @@ public async Task<IActionResult> GetByStreetcodeId([FromRoute] int streetcodeId)
         return HandleResult(await Mediator.Send(new GetFactByStreetcodeIdQuery(streetcodeId)));
     }
 
-    [HttpPut]
```
Comment (yaninavoda) [inline Streetcode/Streetcode.WebApi/Controllers/Streetcode/TextContent/FactController.cs:33]: Why is this deleted?

**Review by yaninavoda — CHANGES_REQUESTED** [review]
Why is ReorderFacts deleted from the FactController?

File: Streetcode/Streetcode.WebApi/Controllers/Streetcode/TextContent/FactController.cs (hunk 1 above)
Comment (OlhaPivniuk) [reply Streetcode/Streetcode.WebApi/Controllers/Streetcode/TextContent/FactController.cs:33]: I put it after UpdateFactDto

Comment (OlhaPivniuk) [reply Streetcode/Streetcode.WebApi/Controllers/Streetcode/TextContent/FactController.cs:33]: I put it after UpdateFactDto to have order Post, Put, Put, Delete (not Put, Post, Put, Delete as we had earlier)

**Review by yaninavoda — APPROVED** [review]
ok

**Pull request merged**